assets/dev/js/editor/utils/container-helper.js

```diff
@@ -72,43 +72,130 @@ export class ContainerHelper {
 	 * @param {string} preset - Preset structure of the sub containers (e.g. `33-66-66-33`).
 	 * @param {Container} target - The target container of the newly created Container.
 	 * @param {Object} options - Additional command options.
+	 * @param {Boolean} [options.createWrapper=true] - Create a wrapper container for the preset.
 	 *
-	 * @return {Container}
+	 * @returns {Container} - Container created on.
 	 */
-	static createContainerFromPreset( preset, target, options = {} ) {
-		const sizes = preset.split( '-' );
-
-		// Map rounded, user-readable sizes to actual percentages.
-		const sizesMap = {
-			33: '33.3333',
-			66: '66.6666',
-		};
-
-		// Create a parent container to contain all of the sub containers.
-		const parentContainer = this.createContainer( {
-				flex_direction: this.DIRECTION_ROW,
-				flex_wrap: 'wrap',
-				flex_gap: {
-					unit: 'px',
-					size: 0, // Set the gap to 0 to override the default inherited from `Site Settings`.
-				},
-			}, target, options );
-
-		// Create all sub containers using the sizes array.
-		// Use flex basis to make the sizes explicit.
-		sizes.forEach( ( size ) => {
-			size = sizesMap[ size ] || size;
-
-			this.createContainer( {
-				flex_direction: this.DIRECTION_COLUMN,
-				width: {
-					unit: '%',
-					size,
-				},
-			}, parentContainer, { edit: false } );
-		} );
+	static createContainerFromPreset( preset, target = elementor.getPreviewContainer(), options ) {
```

Comment:
This function is huge, please extract to smaller functions.

Reply:
Try now.